### PR TITLE
Status for items and uneditable fields

### PR DESCRIPTION
**Pull request by richmondnao**
Finished the Status and Made fields for other status uneditable

### CODE REVIEW

File: src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx [hunk 1]
```diff
@@ -339,6 +342,38 @@ const MissingItemFormEdit = () => {
...
+              {/* Display a chip for items with statuses other than "active" */}
+              {formData.status.toLowerCase() !== 'active' &&
+                formData.status.toLowerCase() !== 'found' && (
+                  <Grid container xs={9.7} className={styles.foundContainer} rowGap={2}>
+                    <Grid container item xs={12} md={6} rowGap={2}>
+                      <Grid item xs={12}>
+                        <Chip
+                          className={styles.largeChip}
+                          sx={{
+                            height: 'auto',
```
Comment (Centerville1) [inline src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx:354]: Can we move height: 'auto' into the CSS largeChip class?  I know the mui override section here needs to be in the sx, but this line doesn't as far as I can tell.

**Review by Centerville1 — CHANGES_REQUESTED** [review]
Overall looks good, just one minor CSS fix, then should be good to go 👍

File: src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx (hunk 1 above)
Comment (richmondnao) [reply src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx]: Sure

**Review by Centerville1 — APPROVED** [review]
LGTM

**Pull request merged**